src/chimera/interfaces/focuser.py

```diff
@@ -26,8 +26,10 @@
 
 FocuserFeature = Enum("TEMPERATURE_COMPENSATION",
                       "ENCODER",
-                      "POSITION_FEEDBACK")
+                      "POSITION_FEEDBACK",
+                      "HEXAPOD")
```

Comment:
I think that this could be more generic, like `CONTROLLABLE_X`, `CONTROLLABLE_Y`, ..., `CONTROLLABLE_W`.

How you specify in a focuser if it can do only axis `z`, or can do `x, y, z`, or all `x, y, z, w`???